packages/amplify-e2e-tests/src/__tests__/function_14.test.ts

```diff
@@ -33,7 +33,7 @@ describe('amplify push function cases:', () => {
     const functionPath = pathManager.getResourceDirectoryPath(projRoot, AmplifyCategories.FUNCTION, funcName);
     const amplifyState = JSONUtilities.readJson<any>(path.join(functionPath, 'amplify.state'));
 
-    expect(amplifyState.scripts.build).toEqual('npm install --no-bin-links --production');
+    expect(amplifyState.scripts.build).toMatch('npm.* install --no-bin-links --production');
```

Comment:
It probably won't ever come up, but this is an unsafe regex to be using. Better to do 'npm(?:\.exe) ...' if we're expecting a windows executable